cms/templates/partials/hero.html

```diff
@@ -1,4 +1,22 @@
-{% load wagtailembeds_tags %}
+{% load wagtailembeds_tags wagtailimages_tags %}
+
+{% block extrahead %}
+  {% if page.background_image %}
+  <style>
+    .header-block {
+      {% image page.background_image fill-1024x768 as background_image %}
+      background-image: url('{{ background_image.url }}');
+    }
+
+    @media screen and (min-width: 1024px) {
```

Comment:
Video is only rendering until I have width > `1200px`, I believe it should render until width is `768px`. Could you please confirm this.

Reply:
I updated the breakpoint to 768px

Reply:
May be there is some other CSS effecting this, I can see video playing under <992px. Also on >992 I can see bg image as well behind the video. Not sure, but I believe we should fix this as well. 
<img width="997" alt="Screen Shot 2019-06-25 at 12 31 50 PM" src="https://user-images.githubusercontent.com/4245618/60078354-35a42c00-9745-11e9-99b2-7817db9d9bc7.png">